### PR TITLE
Feature/only the list example

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Since the ChangePassword screen is still a WIP, this PR only includes the "In a List" example. 
- This should get merged in pretty quickly since this example is in a good spot.

### CODE REVIEW

File: src/pages/form-validation/list/index.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+const useStyles = makeStyles((theme: Theme) => ({
+    containerWrapper: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'start',
+        flex: '1 1 0',
+    },
+    container: {
+        maxWidth: 686,
```
Comment (daileytj) [inline src/pages/form-validation/list/index.tsx:30]: Super minor, but this `maxWidth` seems arbitrary to me despite it being the same width as the design. I commented on Ekta's angular PR with the same comment. I would at least like it to be a factor of 8 to keep in step with the rest of our design system. I ended up using 600pt in react-native. Just my two cents, do with it what you will.

File: src/pages/form-validation/list/index.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        margin: theme.spacing(4),
```
Comment (daileytj) [inline src/pages/form-validation/list/index.tsx:31]: `margin-top` should be 24 so you can just do this:
```suggestion
        margin: theme.spacing(3),
```

File: src/pages/form-validation/list/index.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                                        },
+                                    }}
+                                    onChange={onIpChange}
+                                    variant="filled"
+                                />
+                            }
+                        ></InfoListItem>
+
+                        <InfoListItem
+                            icon={<Timeline />}
```
Comment (daileytj) [inline src/pages/form-validation/list/index.tsx:114]: I'd still like to see that `insights` icon if it's available.

File: src/pages/form-validation/list/index.tsx (hunk 1 above)
Comment (emclaug2) [reply src/pages/form-validation/list/index.tsx:30]: I'm cool with a factor of 8, I'll update it now.

File: src/pages/form-validation/list/index.tsx (hunk 3 above)
Comment (emclaug2) [reply src/pages/form-validation/list/index.tsx:114]: Yeah the insights icon has been an issue, the current version of material-ui/icons we're using in this repo doesn't have the Insights icon available.   I didn't want to update the version just to get this icon so I think Timeline is a good compromise.

File: src/pages/form-validation/list/index.tsx (hunk 1 above)
Comment (emclaug2) [reply src/pages/form-validation/list/index.tsx:30]: I went with 600 as well.

**Review by daileytj — APPROVED** [review]
The green button is all yours @emclaug2

**Comment by emclaug2** [comment]
THANKS!

**Pull request merged**